service/policy/db/migrations/20250703000000_obligations.sql

```diff
@@ -0,0 +1,84 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS obligation_definitions
+(
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    namespace_id UUID NOT NULL REFERENCES attribute_namespaces(id) ON DELETE CASCADE,
+    -- name is a unique identifier for the obligation definition within the namespace
+    name VARCHAR NOT NULL,
+    -- implicit index on unique (namespace_id, name) combo
+    -- index name: obligation_definitions_namespace_id_name_key
+    UNIQUE (namespace_id, name),
```

Comment:
Can you please move these to the end of the create statement so the columns themselves are all defined before the constraints and is consistent with the other migrations